### DISCUSSION TITLE
Can you guys add online terminal

### DISCUSSION THREAD

**Discussion by oyedejioyewole** [vercel/vercel#5188] [Ideas] [answered]
Just like `PythonAnywhere` can you add like terminals online built on like linux for interacting with deployed applications because some developers like me use online environmental variables that aren't available to my local system

If it can't be done then please maybe try adding `ssl` or `ssh` support for deployed applications so that it can be connectable through terminals supporting `ssh` or `ssl` it will aid production development.

**Comment by styfle** [comment, marked as answer]
Hi @LokotamaTheMastermind 

Vercel does not use always-on servers like legacy hosting companies.

Instead, [Serverless Functions](https://vercel.com/docs/serverless-functions/introduction) execute your code and scale on-demand as traffic increases.

Therefore, you wouldn't be able to ssh into the serverless function because there might be multiple instances of it, or zero instances in the case of zero traffic.

We don't plan to add support for such a feature, thanks!

**Reply by oyedejioyewole** [reply]
That's a shame I really hoped that could be possible
